### PR TITLE
Added the task solution

### PR DESCRIPTION
**Pull request by oleksusov**
[DEMO LINK](https://oleksusov.github.io/react_decompose/)

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
```
Comment (TarasHoliuk) [inline src/components/Header/Header.jsx:3]: Add corresponding components to GitHub
image
FYI: `git commit -am 'your message'` doesn't add newly created files. use `git add .` and `git commit -m 'your message'` instead

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
The Demo link doesn't work
Please, make sure you've deployed your app before doing a review request

**Comment by oleksusov** [comment]
Screenshot 2022-11-25 at 11 50 17 PM

**Comment by oleksusov** [comment]
I don't know why this demo link is not working

**Review by loralevitska — CHANGES_REQUESTED** [review]
still not fixed lilnk 
image

Did you ask in fe-chat or Q&A? 
Did you follow the order in which the commands were executed? linter/test everything should go before deployment

**Pull request closed without merging**